src/docdb/nodes.ts

```diff
@@ -0,0 +1,160 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as vscode from 'vscode';
+import * as vm from 'vm';
+import * as path from 'path';
+import { EventEmitter, Event, Command } from 'vscode';
+import { AzureAccount } from '../azure-account.api';
+import { INode, ErrorNode } from '../nodes';
+import { ResourceManagementClient } from 'azure-arm-resource';
+import docDBModels = require("azure-arm-documentdb/lib/models");
+import DocumentdbManagementClient = require("azure-arm-documentdb");
+import { MongoDatabaseNode } from '../mongo/nodes';
+import { MongoCommands } from '../mongo/commands'
+import { DocumentClient } from 'documentdb';
+
+
+export interface IDocDBServer extends INode {
+	getPrimaryMasterKey(): string;
+	getConnectionEndpoint(): string;
+}
+
+export class DocDBServerNode implements INode {
+	readonly contextValue: string = "DocDBServer";
+	readonly label: string;
+
+	readonly collapsibleState = vscode.TreeItemCollapsibleState.Collapsed;
+
+	constructor(private readonly _primaryMasterKey: string, readonly id: string, private readonly _endpoint: string) {
+		this.label = id;
+	}
+
+	get iconPath(): any {
+		return {
+			light: path.join(__filename, '..', '..', '..', '..', 'resources', 'icons', 'light', 'DataServer.svg'),
+			dark: path.join(__filename, '..', '..', '..', '..', 'resources', 'icons', 'dark', 'DataServer.svg')
+		};
+	}
+
+	getPrimaryMasterKey(): string {
+		return this._primaryMasterKey;
+	}
+
+	getEndpoint(): string {
+		return this._endpoint;
+	}
+
+	async getChildren(): Promise<INode[]> {
+		let client = new DocumentClient(this.getEndpoint(), { masterKey: this.getPrimaryMasterKey() });
+		let DocDBServerNodeInstance = this;
```

Comment:
DocDBServerNodeInstance is never used - remove it.

(sidenote: variables should always start with lowercase letters. It helps differentiate from class names, which are always uppercase)